lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1034,8 +1034,8 @@ public function getDateDiffExpression($date1, $date2)
     /**
      * Returns the SQL to add the number of given seconds to a date.
      *
-     * @param string  $date
-     * @param integer $seconds
+     * @param string         $date
+     * @param integer|string $seconds
```

Comment:
No, what I'm saying is creating an `Expr` object or such. Passing around strings here is just going to be causing more and more problems in the future :(